frontend/src/components/crop/Wrapper.tsx

```diff
@@ -62,6 +62,7 @@ export default function DisplayAreaWrapper({
         totalImages={copies}
         currentImageIndex={imageIndex}
         imageIndexSetter={setImageIndex}
+        selection_operations={selectionOperations}
```

Comment:
I think this may be a typo, the GUI shows no image in the plot unless I change it to camel case?
```suggestion
        selectionOperations={selectionOperations}
```

Reply:
Ah sorry, not sure how that one slipped through the cracks